frontend/src/components/Workspace/FlowChart/FlowChartNode/AlgorithmNode.tsx

```diff
@@ -122,7 +122,11 @@ const AlgorithmNodeImple = memo(function AlgorithmNodeImple({
         <Button
           size="small"
           onClick={onClickOutputButton}
-          disabled={status === NODE_RESULT_STATUS.PENDING}
+          disabled={
```

Comment:
@sanglevinh 
Basically, the operation seems to be OK, but as for the OUTPUT button of the node that caused the error as shown in the attachment, the problem does not seem to have been resolved.
Is it possible to address the above? 
(Assume that the judgment is based on the presence or absence of OUTPUT data. If it is difficult, judgment is made based on the presence or absence of errors)

![image](https://github.com/arayabrain/barebone-studio/assets/122338895/e02ecc55-85e6-4917-b13f-a536f700592b)

Reply:
@itutu-tienday i fixed it, pls check it again